frontend/components/admin/rsd-users/RsdLoginList.tsx

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2023 Dusan Mijatovic (dv4all)
+// SPDX-FileCopyrightText: 2023 dv4all
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import {LoginForAccount} from './apiRsdUsers'
+
+export default function RsdLoginForAccount({logins}:{logins:LoginForAccount[]}) {
+  return (
+    <ul className="text-sm">
+      {logins.map(login => {
+        const {id, name, email, home_organisation, provider} = login
+        return (
+          <li key={id}>
+            {name}{email ? `, ${email}`:''}{home_organisation ? `, ${home_organisation}`:''} {provider ? `, ${provider}`:''}
```

Comment:
```suggestion
            {name}{email ? `, ${email}`:''}{home_organisation ? `, ${home_organisation}`:'NULL'} {provider ? `, ${provider}`:''}
```

This would help us to better identify people that login via social IDPs integrated in the AAI.

Reply:
Currently I just show values if these are present without any labels (eg. email, organisation). I not sure that NULL would would have proper meaning without a clear label. Maybe use some other value to clearly indicate that?